payment/src/main/java/com/kyriba/conference/payment/service/PaymentMethodService.java

```diff
@@ -0,0 +1,20 @@
+package com.kyriba.conference.payment.service;
+
+import com.kyriba.conference.payment.api.dto.PaymentMethodDto;
+import com.kyriba.conference.payment.api.dto.PaymentMethodResponse;
+import com.kyriba.conference.payment.api.dto.PaymentMethodUrlUpdateDto;
+import com.kyriba.conference.payment.domain.PaymentMethodType;
+
+import java.util.List;
+
+public interface PaymentMethodService {
+    List<PaymentMethodDto> findAll();
+
+    PaymentMethodResponse createPaymentMethod(PaymentMethodDto paymentMethodDto);
```

Comment:
I thought 'create' method should return an Id. @xpinjection please explain

Reply:
Why id? I use the type as an identifier and this is a wrapper for the type at the moment I can expand it by necessity.

Reply:
Usually when resource is created we return ID from the service and wrap into DTO at API level to use JSON response type.

Reply:
ok. I use a unique type like id. what's wrong?

Reply:
Just meaningless wrapper. :)